lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -10,52 +10,70 @@
 @import '../../scss-shared/nx-input-helpers';
 
 .nx-radio {
-  $circle-width: 16px;
-  $label-margin: 8px;
-  $line-height: 20px;
-
   @include container-horizontal;
   @include regular();
   @include focusable-toggle-input(".nx-radio__outer-circle");
 
   display: inline-block;
-  font-size: 14px;
-  line-height: $line-height;
+  margin-bottom: $nx-spacing-xs;
   position: relative;
   white-space: nowrap;
 
-  &:hover:not(.nx-radio--disabled) {
-    cursor: pointer;
+  &:hover {
+    .nx-radio__outer-circle {
+      stroke: #2c2c2c;
+    }
   }
+}
 
+.nx-radio--disabled {
   .nx-radio__circle {
-    color: $nx-text-color-light;
-    cursor: pointer;
-    height: $line-height;
-    margin-right: $label-margin;
-    width: $circle-width;
-    vertical-align: top;
-
-    &:last-child {
-      // due to specificity issues the `container-horizontal` mixin on nx-radio isn't successfully setting this
-      margin-right: 0;
-    }
+    cursor: default;
   }
 
-  .nx-radio__outer-circle {
-    stroke: currentColor;
+  .nx-radio__content {
+    color: #949494;
   }
 
-  .nx-radio__inner-circle {
-    fill: currentColor;
+  &, &:hover {
+    .nx-radio__outer-circle {
+      fill: #e9e9e9;
+      stroke: #949494;
+    }
   }
+}
+
+.nx-radio__circle {
+  border-radius: 8px;
+  cursor: pointer;
+  fill: none;
+  height: 24px;
+  margin-right: $nx-spacing-md;
+  width: 16px;
+  vertical-align: top;
+}
+
+.nx-radio__outer-circle {
+  stroke: #d1d6f0;
+}
+
+.nx-radio__inner-circle {
+  fill: #fff;
+  stroke: #045dd1;
+}
+
+.nx-radio__input {
+  margin: 0;
```

Comment:
What's this for?

Reply:
> What's this for?

Overrides browser agent styling

Reply:
> Overrides browser agent styling

But this component uses absolute positioning, so I'm not finding a way in which it's margin matters

Reply:
> > Overrides browser agent styling
> 
> But this component uses absolute positioning, so I'm not finding a way in which its margin matters

There was something I was doing where the margin was an issue, but even if I'm mistaken I see no harm in removing the user agent margin